inst/pages/miaverse.qmd

```diff
@@ -121,9 +121,11 @@ The following DA methods support `(Tree)SummarizedExperiment`.
 - [ALDEx2](https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html) for differential abundance analysis
 
 ### Other relevant packages
-
-- [LinDA](https://github.com/zhouhj1994/LinDA)
-- [vegan](https://vegandevs.github.io/vegan/)
+- [MGnifyR](http://github.com/beadyallen/mgnifyr) for accessing and processing MGnify data in R
```

Comment:
I think this link is outdated, the new one is under EBI. CHeck this

Reply:
Thanks